styles.css

```diff
@@ -58,6 +56,53 @@ a {
   display: none;
 }
 
+ul {
+  /* removes padding on ul lists */
+  padding: 0;
+  list-style: none;
+}
+
+.lets-connect a {
+  text-decoration: none;
+  color: #6070ff;
+}
+
+.modal {
+  top: 10%;
+  position: relative;
+  width: 100%;
+  z-index: -1;
+  height: 100%;
+  padding-top: 20px;
+}
+
+.hidden {
+  display: none;
+}
+
+.overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  height: 100vh;
+  background-color: #6070ff;
+  opacity: 0.9;
+  z-index: 10;
```

Comment:
### Your Menu :red_circle:
![image](https://user-images.githubusercontent.com/93039359/208646231-a58c6adb-8e99-4d0f-8daf-8e01e73f7955.png)

### FIgma Menu :green_circle:
![image](https://user-images.githubusercontent.com/93039359/208646583-9c7ade24-d081-4092-8d7e-ffd211ab6bad.png)

- [ ] You can see that there is blur effect in the Figma template but in your design i cannot see it 🤔 Kindly follow the Figma template to make your project professional 😊
- [ ] You can see in the Figma image that there is a black bar at the end of Figma Menu 👀 Kindly add that black bar in your menu to make your design look similar to the Figma template 👍